base/timing.jl

```diff
@@ -144,7 +149,8 @@ end
 
 A macro to execute an expression, printing the time it took to execute, the number of
 allocations, and the total number of bytes its execution caused to be allocated, before
-returning the value of the expression.
+returning the value of the expression. If the time reported includes time spent doing 
```

Comment:
Maybe just

```suggestion
returning the value of the expression. Eventual time spent garbage collecting (gc) or compiling is shown as a percentage.
```

The literal `%` feels a bit surprising in running text.

Reply:
Agreed. "Eventual" didn't feel quite right so went with "Any"